frontend/packages/dev-console/src/test/pipeline-data.ts

```diff
@@ -75,13 +277,15 @@ export const pipelineTestData: PipelineTestData = {
           ],
         },
         status: {
+          completionTime: '2019-12-10T11:18:38Z',
+          pipelineSpec: pipelineSpec[PipelineExampleNames.SIMPLE_PIPELINE],
           conditions: [
             {
               lastTransitionTime: '2019-09-12T20:38:01Z',
               message: 'All Tasks have completed executing',
               reason: 'Succeeded',
-              status: '',
-              type: 'Running',
+              status: 'Unknown',
+              type: 'Succeeded',
```

Comment:
Interesting, this is `DataState.IN_PROGRESS`, you have changed it to completed... I think you're mixing data points here.

Reply:
reverted back.